### DISCUSSION TITLE
Is there more information on MVVMCross (soon)?

### DISCUSSION THREAD

**Discussion by anose001** [unoplatform/uno#17614] [Q&A] [answered]
The Uno Github page states "MVVMCross (soon)" over here [link](https://github.com/unoplatform/uno#:~:text=SkiaSharp.Extended-,MVVMCross%20(soon),-ReactiveUI%20Official). 

I have not been able to find a lot of information what this means as MVVMCross has been around for a while. Any more information on this?

**Comment by MartinZikmund** [comment, marked as answer]
That is indeed outdated information. We are currently not working on MvvmCross support, but feel free to upvote https://github.com/unoplatform/uno/issues/1250, so that it gets prioritized earlier

**Reply by morning4coffe-dev** [reply]
@MartinZikmund I think we should adjust the README and maybe overhaul it a little as well.

**Reply by anose001** [reply]
@MartinZikmund, thanks Martin, is there a recommended approach for cross-platform mvvm dev other than the default Uno approach?

**Reply by morning4coffe-dev** [reply]
@anose001 By default, when you choose the MVVM in the Wizard, we actually use the `CommunityToolkit.Mvvm` (https://learn.microsoft.com/en-us/dotnet/communitytoolkit/mvvm/) from Microsoft and I would say that is our most recommended approach. Is there something specific you are missing with this approach?

**Reply by anose001** [reply]
@morning4coffe-dev, I was just curious on how to approach this in general, thanks morning4coffe
